plugins/fireblocks/README.md

```diff
@@ -0,0 +1,36 @@
+# hardhat-fireblocks-signer
```

Comment:
can this be refactored into its own package? I was thinking we could do something similar for other hardhat plugin-like functionality we've added

Reply:
Yes that was my intention in structuring it this way.  Figured it would be simpler to get it working inline first and then promote it into a separate package.